### ISSUE TITLE
upload `dhall-kubernetes-generator` on hackage

### ISSUE THREAD

**Issue by mikefaille** [arianvp/dhall-kubernetes#135]
Dear maintainers,

I want use `dhall-kubernetes-generator` on Nixos. I think it will be better integrated for my workflow if we upload it here `https://hackage.haskell.org/upload` first.

Thank you for your support.

**Comment by TristanCacqueray** [comment]
@mikefaille note that the `dhall-kubernetes-generator` is currently being refactored as a standalone `openapi-to-dhall` command in the `dhall-haskell` project.

@Gabriel439 I ran out of time for that, and my nix Fu is not that great... Can we have a derivation from `dhall-haskell` to replace the current code in `dhall-kubernetes` by the new dhall-openapi package? And could @mikefaille use the same derivation directly?

**Comment by mikefaille** [comment]
Allright then, as long I can get `openapi-to-dhall ` from hackage, I can package it easily in Nixos. It would be a pleasure !

For know, `openapi-to-dhall ` is not available on hackage. I propose to publish this package/lib here.

**Comment by Gabriella439** [comment]
@mikefaille: Yes, we will publish `dhall-openapi` to Hackage soon-ish.  My guess is that we will publish it as one of the next two releases (with one release every month)
